tix_clock/tix_clock.ino

```diff
@@ -7,19 +7,27 @@ const int LATCH_PIN = 3;
 const int CLOCK_PIN = 4;
 const int NUMBER_OF_OUTPUT_PINS = 3;
 int OUTPUT_PINS[] = {DATA_PIN, LATCH_PIN, CLOCK_PIN};
+int hours = 25;
+int tensHours;
+int unitsHours;
+
 
 void SetInputPins(int[], int);
 void SetOutputPins(int[], int);
 
 void setup() 
 {
+    Serial.begin(9600);
     SetInputPins(INPUT_PINS, NUMBER_OF_INPUT_PINS);
     SetOutputPins(OUTPUT_PINS, NUMBER_OF_OUTPUT_PINS);
 }
 
 void loop() 
 {
-
+    tensHours = GetTensFromNumber(hours);
```

Comment:
please remove  this from the void loop. The Pull Request description said that you only added the function definitions